bitbots_head_mover/config/head_config.yml

```diff
@@ -0,0 +1,212 @@
+move_head:
+  # Max values for the head position
+  max_pan: {
+    type: double_array,
+    default_value: [-2.35, 2.35],
+    description: "Max values for the head position (in radians)",
+    validation: {
+          fixed_size<>: 2
+        }
+  }
+  max_tilt: {
+    type: double_array,
+    default_value: [-1.2, 0.2],
+    description: "Max values for the head position (in radians)",
+    validation: {
+          fixed_size<>: 2
+        }
+  }
+
+  look_at:
+      tilt_speed: {
+        type: double,
+        default_value: 6.0,
+      }
+      pan_speed: {
+        type: double,
+        default_value: 6.0
+      }
+  # Search pattern for ball
+  search_pattern:
+      # Speed search pattern
+      tilt_speed: {
+        type: double,
+        default_value: 3.0
+      }
+      pan_speed: {
+        type: double,
+        default_value: 3.0
+      }
+      # Max values for the search pattern
+      pan_max: {
+        type: double_array,
+        default_value: [135.0, -135.0],
+        description: "Maximum pan values for the search pattern (in degrees)",
+        validation: {
+          fixed_size<>: 2
+        }
+      }
+      tilt_max: {
+        type: double_array,
+        default_value: [-10.0, -60.0],
+        description: "Maximum tilt values for the search pattern (in degrees)",
+        validation: {
+          fixed_size<>: 2
+        }
+      }
+      # Number of scan lines for the search pattern
+      scan_lines: {
+        type: int,
+        default_value: 2,
```

Comment:
Add minimum number of scanlines

Reply:
minimum number of scanlines should be 1 right?

Reply:
Yep